qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -887,6 +947,9 @@
         <height>22</height>
        </size>
       </property>
+      <property name="toolTip">
+       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Override display of linear data in gamma space&lt;/p&gt;&lt;p&gt;See FAQ on &amp;quot;Gamma display of linear data&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
I'm not sure why this tooltip is all in html, it doesn't seem to contain any special formatting? Can it be converted back to plaintext?